sdk/src/test/tutorial/java/parity/FeatureLookthrough.java

```diff
@@ -0,0 +1,99 @@
+package parity;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+public class FeatureLookthrough {
+
+    String annotations;
+
+    /**
+     * Scans all classes accessible from the context class loader which belong
+     * to the given package and subpackages.
+     *
+     * @param packageName The base package
+     * @return The classes
+     * @throws ClassNotFoundException is thrown when no classes are found
+     * @throws IOException            is thrown when the input is not acceptable
+     */
+    private Iterable<Class> getClasses(String packageName) throws ClassNotFoundException, IOException, URISyntaxException {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        String path = packageName.replace('.', '/');
```

Comment:
I think you need to use `File.PathSeparator` rather than `/` so it works on Windows